### ISSUE TITLE
Latest version of html5lib throws TypeError with example code

### ISSUE THREAD

**Issue by ouroborus** [edsu/microdata#33]
Latest version of html5lib (v0.999999999 (nine 9s)) throws TypeError when using example code:

```
Traceback (most recent call last):                                                           
  File "C:\Users\landonjx\Documents\scrape\scrape.py", line 10, in <module>                  
    items = microdata.get_items(urllib.urlopen('http://www.telegraph.co.uk/'))               
  File "C:\Python27\lib\site-packages\microdata.py", line 21, in get_items                   
    tree = parser.parse(location, encoding=encoding)                                         
  File "C:\Python27\lib\site-packages\html5lib\html5parser.py", line 235, in parse           
    self._parse(stream, False, None, *args, **kwargs)                                        
  File "C:\Python27\lib\site-packages\html5lib\html5parser.py", line 85, in _parse           
    self.tokenizer = _tokenizer.HTMLTokenizer(stream, parser=self, **kwargs)                 
  File "C:\Python27\lib\site-packages\html5lib\_tokenizer.py", line 36, in __init__          
    self.stream = HTMLInputStream(stream, **kwargs)                                          
  File "C:\Python27\lib\site-packages\html5lib\_inputstream.py", line 151, in HTMLInputStream
    return HTMLBinaryInputStream(source, **kwargs)                                           
TypeError: __init__() got an unexpected keyword argument 'encoding'                          
```
v0.9999999 (seven 9s) works fine.

**Comment by joke2k** [comment]
Hi @ouroborus probably the API of html5lib is changed.
With this commit https://github.com/html5lib/html5lib-python/commit/fc9f63be7cf2530f28b2e0bf8b5a767e2ac2c8be they had changed the `encoding` with `transport_encoding`.

My PR resolves that, and it has just been merged

**Comment by ouroborus** [comment]
Thank you much.

**Comment by edsu** [comment]
Seems we now have a problem on Python3....

https://travis-ci.org/edsu/microdata/jobs/147837942

**Comment by edsu** [comment]
In https://github.com/edsu/microdata/commit/bff9c499f4ece7757d0a8bcabbb59f63b68d3677 I've adjusted the code to only use `transport_encoding` if `encoding` is supplied.

**Comment by edsu** [comment]
Just released as v0.6.0 to PyPI.
